src/main/java/duke/util/Ui.java

```diff
@@ -0,0 +1,22 @@
+package duke.util;
+/**
+ * Ui class to print display items
+ */
+public class Ui {
```

Comment:
Would you consider delegating more of the display items, such as adding and list messages, to the Ui?

Reply:
I will look into this. Previously, the changes to be made to the UI class to return strings instead of printing to the console made it a little confusing. However, this is indeed a great suggestion. Thanks for the input!